addons/mail/models/mail_alias.py

```diff
@@ -229,11 +229,22 @@ def open_parent_document(self):
         }
 
     def _get_alias_bounced_body_fallback(self, message_dict):
-        return Markup(
-            _("""<p>Hi,<br/>
-The following email sent to %s cannot be accepted because this is a private email address.
-Only allowed people can contact us at this address.</p>""")
-        ) % self.display_name
+        if self.alias_contact == 'partners':
+            group_description = _('addresses linked to registered partners')
+        elif self.alias_contact == 'followers':
+            group_description = _('a specific list of email addresses')
+        else:
+            group_description = _('addresses linked to registered employees')
```

Comment:
I'm guessing this is to avoid an override in the hr module?
Doesn't seem very modular but it would be annoying to override only the "group_description" thing, unless you make a separate method.

Seems there is also a "everyone" value for alias_contact, that will then print something related to employees, is that ok?

Reply:
Indeed, I was lazy but you caught me :D ok then, made a small tool method and an override. About "everyone": actuallyyou should not bounce if everyone is accepted. If this is the case due to some strange reason, it now falls back into "specific list of email addresses", as we cannot say "everyone can write but not you, don't know why" ^^ .